corehq/apps/users/tasks.py

```diff
@@ -86,7 +86,7 @@ def make_link(url, i):
                 'The following groups have no name. '
                 'Please name them before continuing: {}'
             ),
-            mark_safe(', '.join(group_links))
+            mark_safe(', '.join(group_links))  # nosec: no user input
```

Comment:
I wonder if we could take advantage of the fact that `make_link` returns safe strings and actually still use `format_html` here still. Like

```python
format_html(', '.join(['{}'] * len(group_links)), group_links)
```

I think even I don't like my suggestion so totally feel free to ignore it. Just noting here in case it sparks you to think of a more elegant convention we could use in a situation like this. The only concrete benefit of the line above is that future changes in `group_links` wouldn't be able to inadvertently introduce XSS vulnerability, and that's admittedly an overall minor consideration at this point.

Reply:
Is [format_html_join](https://docs.djangoproject.com/en/3.1/ref/utils/#django.utils.html.format_html_join) similar to what you had in mind? I was on the verge of refactoring this to use `format_html_join` and then decided it probably wasn't worth the effort over just tagging it with `# nosec`